### PR TITLE
[T3A2][T10-A2] Low Yong Cheng

### PR DESCRIPTION
**Pull request by yongchenglow**
Added the Encapsulation to CommandResult

### CODE REVIEW

**Comment by yongchenglow** [comment]
Ready For Review

File: src/seedu/addressbook/commands/CommandResult.java [hunk 1]
```diff
@@ -11,18 +11,18 @@
 public class CommandResult {
 
     /** The feedback message to be shown to the user. Contains a description of the execution result */
-    public final String feedbackToUser;
+    private static String feedbackToUser;
```
Comment (nirandiw) [inline src/seedu/addressbook/commands/CommandResult.java]: What is the point of making this static? Not necessary.

File: src/seedu/addressbook/commands/CommandResult.java [hunk 2]
```diff
@@ -11,18 +11,18 @@
...
 
     /** The list of persons that was produced by the command */
     private final List<? extends ReadOnlyPerson> relevantPersons;
 
-    public CommandResult(String feedbackToUser) {
-        this.feedbackToUser = feedbackToUser;
+    public CommandResult(String feedback) {
+        setFeedbackToUser(feedback);
```
Comment (nirandiw) [inline src/seedu/addressbook/commands/CommandResult.java]: What is the point of this set method? You need to simply assign that value to a private class variable. a set method is not necessary.

**Comment by nirandiw** [comment]
Some comments added. Please close the PR after reading comments.

**Pull request closed without merging**